eslint.yaml

```diff
@@ -21,7 +21,7 @@ rules:
   no-trailing-spaces: 2
   space-after-keywords: 2
   space-before-blocks: 2
-  space-before-function-paren: 2
+  space-before-function-paren: 1
```

Comment:
Relaxed this to a warning on the main, and turned it off for tests.
But we can go all the way with it @jeff-phillips-18 @dtaylor113 @amarie401 ?

Reply:
I say remove it completely.

Reply:
cannnnn do!